zetaclient/evm_client.go

```diff
@@ -75,6 +75,7 @@ type EVMChainClient struct {
 	txWatchList               map[ethcommon.Hash]string
 	Mu                        *sync.Mutex
 	db                        *gorm.DB
+	outTxPendingTransaction   map[string]*ethtypes.Transaction
 	outTXConfirmedReceipts    map[string]*ethtypes.Receipt
 	outTXConfirmedTransaction map[string]*ethtypes.Transaction
```

Comment:
We should use plural
`outTxPendingTransactions` and `outTXConfirmedTransactions`

Reply:
[commit](https://github.com/zeta-chain/node/pull/1518/commits/0b42c7b456b475c9cf0e40aa62a2875d5da966ff)